include/avif/avif.h

```diff
@@ -544,7 +544,7 @@ typedef struct avifDecoder
     // Set this via avifDecoderSetSource().
     avifDecoderSource requestedSource;
 
-    // The current decoded image, owned by the decoder. Can be NULL if the decoder hasn't run or has run
+    // The current decoded image, owned by the decoder. Is invalid if the decoder hasn't run or has run
     // out of images. The YUV and A contents of this image are likely owned by the decoder, so be
```

Comment:
Alternatively, I can change the second sentence to "Can be NULL if the decoder hasn't run." Please let me know which one you prefer.

In fact, decoder->image is invalid after any error, not just AVIF_RESULT_NO_IMAGES_REMAINING. So the proposed comment can still be improved.